src/constants.rs

```diff
@@ -2,13 +2,13 @@ use crate::prelude::*;
 use alloy_primitives::U256;
 use num_bigint::Sign;
 
-#[derive(Clone, Copy, Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Let's keep it. There shouldn't be much cost if any.

Reply:
will do that